paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -638,6 +713,25 @@ internal class DefaultFlowController @Inject internal constructor(
         is PaymentResult.Failed -> PaymentSheetResult.Failed(throwable)
     }
 
+    private fun getPaymentAppearance(): PaymentSheet.Appearance {
+        return viewModel.previousConfigureRequest?.configuration?.appearance ?: PaymentSheet.Appearance()
```

Comment:
We should get this from state, rather than the `previousConfigureRequest`. So it should be `viewModel.state.config`